fiss_api_addons.py

```diff
@@ -82,3 +82,9 @@ def import_workspace_attributes_TSV(namespace, workspace, attrs):
     uri = "workspaces/{0}/{1}/importAttributesTSV".format(namespace, workspace)
 
     return fapi.__post(uri, headers=headers, data=body)
+
+def get_workspace_cloudPlatform(namespace, name):
+    ''' get cloud platform of workspace'''
```

Comment:
nit: use double quotes, and remove initial space before text. (same goes for line 70 in the other file)